homeassistant/components/generic/config_flow.py

```diff
@@ -109,6 +109,20 @@ def build_schema(
     return vol.Schema(spec)
 
 
+def build_schema2(user_input: dict[str, Any] | MappingProxyType[str, Any]):
+    """Create schema for conditional 2nd page specifying stream content_type."""
+    return vol.Schema(
+        {
+            vol.Required(
+                CONF_CONTENT_TYPE,
```

Comment:
Why is content type necessary? I don't see it in the YAML config? https://www.home-assistant.io/integrations/generic

Reply:
It used to be in the docs but I removed it for 2022.4 because I thought it could be determined automatically.
https://github.com/home-assistant/home-assistant.io/pull/14841/files

Now having to add it back in because of #69128.  Evidently the camera object complains if it doesn't have a content_type, and it can't be determined automatically if only a stream is specified.

Reply:
> and it can't be determined automatically if only a stream is specified.

Rather that has not been implemented (yet).  But I intend to.

Reply:
Between step 1 and 2 we can try to determine the content type. If we can't determine, ask the user for content type.

Reply:
Yes, this is how it is now implemented.